packages/babel-core/test/api.js

```diff
@@ -115,11 +115,11 @@ describe("api", function () {
   });
 
   it("exposes the resolvePlugin method", function() {
-    assert.equal(babel.resolvePlugin("nonexistent-plugin"), null);
+    assert.throws(() => babel.resolvePlugin("nonexistent-plugin"));
```

Comment:
This test is `"api exposes the resolvePlugin method"`, but wouldn't this change stop verifying that the `resolvePlugin` method is actually exposed in the API? Because if the method wasn't exposed, this would throw `TypeError: babel.resolvePreset is not a function`, which would pass the `assert.throws` assertion, right?